en/airframes_experimental/traxxas_stampede.md

```diff
@@ -0,0 +1,132 @@
+
+> **Warning** It is **HIGHLY RECOMMENDED** to set the ESC in training mode (see Traxxas Stampede Manual) so to reduce the power to 50%.
+
+
+## Output connections
+
+| PWM Output | Rate | Actuator |
+| -- | -- | -- |
+| MAIN1 | 50 Hz | Not connected |
+| MAIN2 | 50 Hz | Steering servo |
+| MAIN3 | 50 Hz | Not Connected |
+| MAIN4 | 50 Hz | ESC input |
+| MAIN5 | 50 Hz | Not Connected |
+| MAIN6 | 50 Hz | Not Connected |
+| MAIN7 | 50 Hz | Not Connected |
+| MAIN8 | 50 Hz | Not Connected |
+
+## Simulation
+
+To test the algorithms, a simulation was used. This can be ran by running 
+
+ ```sh
+ make posix gazebo-rover
+ ``` 
+
+ from the Firmware folder. More details on how to do it can be found in section [Gazebo Simulation](../simulation/gazebo.md).
+
+## Usage
+At the current time, PX4 only supports MISSION and MANUAL modes when a RC remote is connected. To use the mission mode, first upload a new mission to the vehicle with QGC. Then, BEFORE ARMING, select `MISSION` and then arm. 
+
+> **Warning** It is **_VERY IMPORTANT_** to do a mission composed **_ONLY_** by normal waypoints (i.e. **_NO TAKEOFF WAYPOINTS_**) and it is crucial to **_SET THE WAYPOINT HEIGHT OF EVERY WAYPOINT TO 0_** for a correct execution. Failing to do so will cause the rover to continuously spin around a waypoint.
+
+
+A correct mission setup looks as follows
+
+![mission](../../assets/airframes/experimental/stampede/correct_mission.jpg)
+ 
+Since at the current state of development there is no obstacle avoidance, it is highly recommended to test missions in the simulator first to get acquainted with the software usage.
+
+## Software location
+
+As it still is a work in progress, the code can be obtained from a personal repository currently being considered for merging in PRs [7139](https://github.com/PX4/Firmware/pull/7139) and [7175](https://github.com/PX4/Firmware/pull/7175). 
+
+```sh
+cd ~ 
+mkdir src
+cd src
+git clone -b pr-rover-rebase git@github.com:marcozorzi/Firmware.git
+cd Firmware
+git submodule update --init --recursive
+```
+
+## Building the firmware
+To build the firmware it is necessary to use the terminal (QGC is not supported yet). This can be done by following the instructions in [Building the Code section](../setup/building_px4.md) and uploading by running
+
+```sh
+cd ~/src/Firmware
+make px4fmu-v2_default upload
+```
+
+## Parameters
+
+The current software provides an attitude controller and a position controller. These build on top of the fixed wing framework to move an Ackerman steered vehicle following GPS coordinates (real or fake). In order to this, 2 PIDs were implemented, one for the steering and the other for the speed. 
+These can be started by setting the following general parameters, which can be set with QGC (instructions [here](https://docs.qgroundcontrol.com/en/SetupView/Parameters.html)):
+* SYS_AUTOSTART 50002
+* MAV_TYPE 10
```

Comment:
I see that the 'airframe' config already sets the `MAV_TYPE` and the tuning params. So it's not necessary to add them here. But I think it's useful to describe which params can be tuned & how.